src/components/UI/select/Select.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import s from './Select.module.scss';
+import { IOptions } from '../../../types/ITypes';
+
+interface IProps {
+  title: string;
+  options: IOptions[];
+  onChange: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
I think far all such handlers returning void will be enough
onChange: () => void
onBlaBla: () => void